company-profiles/gft.md

```diff
@@ -0,0 +1,36 @@
+# GFT
+
+## Company blurb
+
+GFT Technologies SE (GFT) is a business and technology consultancy trusted by the world's leading financial institutions to solve their business challenges. GFT responds to the constant current regulatory change and continuously innovates to meet the demands of the digital revolution. GFT combines advice, creativity and technology with innovation and specialized knowledge of the financial sector to transform the businesses of its clients.
+
+## Company size
+
+GFT has +10.000 employees over Asia, Europe, Latin America and North America.
+
+## Remote status
+
+Full-remote and "work-from-home" policies are common across all regions where GFT has presence.
+
+## Region
+
+Asia, Europe, Latin America and North America.
+
+## Region Details
```

Comment:
Can this be changed to bolder, please? There are automated tests that fail when there are other unexpected headers like this

Reply:
Sure, I forgot that Region Details/examples go in another level. Sorry :)